lib/src/Base/Graph/Pie.cxx

```diff
@@ -232,6 +236,29 @@ void Pie::checkData(const Point & data) const
   }
 }
 
+void Pie::setData(const Point & data)
+{
+  checkData(data);
+
+  // reject too small values
+  const UnsignedInteger size = data.getSize();
+  Point filteredData;
+  Description filteredLabels;
+  const Scalar epsilon = ResourceMap::GetAsScalar("Pie-SmallValue");
+  for (UnsignedInteger i = 0; i < size; ++ i)
+    if (data[i] > epsilon)
+    {
+      filteredData.add(data[i]);
+      if (labels_.getSize())
+        filteredLabels.add(labels_[i]);
+    }
+    else
+      LOGWARN(OSS() << "removed the value " << data[i] << " from pie chart as it is too small");
```

Comment:
```
"... with label " << labels_[i]
```
We could also aggregate all the labels associated to a total value less than epsilon and put a unique label made of all these labels, in order to show on the piechart that they contribute to nothing. Otherwise, the message will be silently ignored in a notebook session...

Reply:
ok, dropped that commit, I'll open an issue instead